### PR TITLE
feat(component_interface_utils): change service log output

### PR DESCRIPTION
**Pull request by isamu-takagi**
## Description

Change service log output from console to topic. Related to https://github.com/tier4/tier4_autoware_msgs/pull/58.
Trying to replay a service from rosbag for AD API debugging.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: common/component_interface_utils/CMakeLists.txt [hunk 1]
```diff
@@ -3,5 +3,5 @@ project(component_interface_utils)
 
 find_package(autoware_cmake REQUIRED)
 autoware_package()
-
+ament_export_dependencies(tier4_system_msgs)
```
Comment (kenji-miyake) [inline common/component_interface_utils/CMakeLists.txt:6]: I think it's automatically exported by `ament_auto_package`.
https://github.com/ament/ament_cmake/blob/799183ab9bcfd9b66df0de9b644abaf8c9b78e84/ament_cmake_auto/cmake/ament_auto_package.cmake#L58

```suggestion
```

Comment (isamu-takagi) [reply common/component_interface_utils/CMakeLists.txt:6]: I thought so too, but actually the build fails without this.

Comment (kenji-miyake) [reply common/component_interface_utils/CMakeLists.txt:6]: Hmm, I see. :thinking: 
Okay, thanks!

**Comment by kenji-miyake** [comment]
> Trying to replay a service from rosbag for AD API debugging.

Related: https://github.com/ros2/rosbag2/issues/773#issuecomment-1145209596

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**